api/v1/paths/games/{gameId}/players/{playerId}.js

```diff
@@ -0,0 +1,28 @@
+const appRoot = require('app-root-path');
+
+const playerDao = require('../../../../db/oracledb/player-dao');
+
+const { errorBuilder, errorHandler } = appRoot.require('errors/errors');
+const { openapi: { paths } } = appRoot.require('utils/load-openapi');
+
+/**
+ * @summary Get games a specific member is now playing
+ */
+const get = async (req, res) => {
+  try {
+    const { gameId } = req.params;
+    const { playerId } = req.params;
+    const result = await playerDao.getPlayersByGameIdAndPlayerId(gameId, playerId);
+    if (!result) {
+      errorBuilder(res, 404, 'A game / player with the specified ID was not found.');
```

Comment:
Could you improve the error message here by specifying which resource wasn't found?